unlock-app/src/components/interface/locks/Settings/forms/CreditCardCustomPrice.tsx

```diff
@@ -141,25 +172,43 @@ export default function CreditCardCustomPrice({
             }}
           />
           {toggleActive && (
-            <Input
-              type="number"
-              step="any"
-              disabled={disabled}
-              description="Set a fixed price in USD that is charged for card payments."
-              error={errors?.creditCardPrice?.message}
-              {...register('creditCardPrice', {
-                required: {
-                  value: toggleActive,
-                  message: 'This field is required.',
-                },
-                min: {
-                  value: CREDIT_CARD_MIN_USD_PRICE,
-                  message: `Price is too low for us to process credit cards. It needs to be at least ${parseFloat(
-                    `${CREDIT_CARD_MIN_USD_PRICE}`
-                  ).toFixed(2)}. `,
-                },
-              })}
-            />
+            <div className="grid gap-2 md:grid-cols-2">
+              <div className="flex flex-col gap-1">
+                <span>Price</span>
+                <Input
+                  type="number"
+                  step="any"
+                  disabled={disabled}
+                  description="Set a fixed price in USD that is charged for card payments."
+                  error={errors?.creditCardPrice?.message}
+                  {...register('creditCardPrice', {
+                    required: {
+                      value: toggleActive,
+                      message: 'This field is required.',
+                    },
+                    min: {
+                      value: CREDIT_CARD_MIN_USD_PRICE,
```

Comment:
I wonder what happens if the user sets 0.1 EUR for example?

Reply:
in this case, the user puts `0.1` will show the error, we only consider price > `0.5` here, 

but let me refactor and add this: 

https://stripe.com/docs/currencies

Reply:
Yes that's exactly what we need!